### ISSUE TITLE
logging: gRPC status should be extracted from stream_info

### ISSUE THREAD

**Issue by Shikugawa** [envoyproxy/envoy#10296]
It seems that we can't extract `grpc-status` from stream_info directly now. Any other status-related info (e.g. response code) can extract from it but that not. I think that is should be extracted from that information pack. Do you think about it?

**Comment by alyssawilk** [comment]
cc @lizan

**Comment by lizan** [comment]
Yeah it should be and I think we already doing that because the gRPC status access log filter operates on that, no?

**Comment by Shikugawa** [comment]
Seems valid. What you are saying is `GrpcAccessLogger`, aren't you? I think that grpc-stream info is not extracted directly in this case. (To get this information, it seems that we should set `grpc-status` to `additional_response_trailers_to_log`) 
It is not needed now but may be required in the future, is it? If not, it is strange that we can extract response code from it but grpc-status not.
